File: Homeworks/01. C# Fundamentals/03. Operators-and-Expressions/04. Rectangles/Rectangles.cs
using System;

class Rectangles
{
    static void Main()
    {
        double widht = double.Parse(Console.ReadLine());
        double height = double.Parse(Console.ReadLine());
        double area = widht * height;
        double perimeter = (widht + height) * 2;
        Console.WriteLine("{0:0.00} {1:0.00}", area, perimeter);
    }
}

File: Homeworks/06. JS Fundamentals/04. Operators and Expressions/03. Rectangles/Rectangles.js
function Rectangles (args) {
	var width, height, perimeter, area;
  	width = args[0];
  	height = args[1];
	area = width * height;
	perimeter = (2 * width) + (2 * height);
	console.log(area.toFixed(2), perimeter.toFixed(2))
}